### PR TITLE
Landing page navigation bar. 

### PR DESCRIPTION
**Pull request by O-Osman1618**

### CODE REVIEW

File: tailwind.config.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      lg: "1024px",
+      xl: "1280px",
+      xs: "480px",
+      xxl: "1920px",
+    },
+    fontFamily: {
+      primary: "var(--font-jetBrainsMono)",
+    },
+    extend: {
+      colors: {
```
Comment (O-Osman1618) [inline tailwind.config.ts:30]: This is crap for now, we need to re-visit.

**Comment by O-Osman1618** [comment]
trying webhook notification.

**Comment by O-Osman1618** [comment]
Test github webhook to discord.

**Comment by Hamza-AlSalkini** [comment]
Nice one, I was wondering if we want to use a component provider like MUI, Chakra UI, Tailwind. 
MUI is nice, it has finished components like buttons, scrolls, and massive of finished components that we can use in the project without needing to design everything by ourselves.

**Pull request merged**